.github/workflows/deploy_nightly.yml

```diff
@@ -63,7 +63,7 @@ jobs:
       # Pre-mode is exited if case we're in it, otherwise it has no effect
       - name: prepare nightly release
         run: |
-          yarn changeset pre exit
+          yarn changeset pre exit || true
```

Comment:
I think I remember `||:` being a canonical slightly better approach for some reason? 🤷

Reply:
I think more people will know what this means 😅